1.8/usage/haproxy.md

```diff
@@ -0,0 +1,77 @@
+---
+post_title: Configuring HTTP Proxy in Front of DC/OS
+nav_title: Configure HTTP Proxy
+menu_order: 61
+---
+
+You can configure a secure communication channel between HAProxy and the DC/OS Admin Router, using a custom hostname and port.
```

Comment:
Well, no, it's generally valid for Open DC/OS, too. However, certificate verification will not work, because there is no CA certificate to verify against (unless people inject their own certificate for Admin Router). 

Should we provide an alternative (outcommented) upstream server definition line that will not require certificate verification?

CC @alberts